test/protocol/clients/BosonVoucherTest.js

```diff
@@ -1861,6 +1865,19 @@ describe("IBosonVoucher", function () {
       const tokenURI = await bosonVoucher.tokenURI(1);
       expect(tokenURI).eq(metadataUri);
     });
+
+    it("should return the correct tokenURI when token is preminted", async function () {
```

Comment:
Can you please add another test, where token is preminted and then transferred. Just to make sure it then behaves as normal and get the tokenURI.

Reply:
added